public/locales/ar/translation.json

```diff
@@ -166,8 +166,9 @@
       }
     ],
     "buttons": {
-      "opportunitiesBtn": "جميع الفرص",
-      "blogBtn": "جميع المدونات"
+      "opportunitiesBtn": "جميع الفرص {{arrow}}",
+      "blogBtn": "جميع المدونات {{arrow}} ",
+      "arrow": "&#8592;"
```

Comment:
i dont think you have to do this arrow thing. if you just put it inline in the html it should handle RTL correctly next to it

Reply:
i mean arrow does not have to be in the translations

the word "arrow" is not being translated, therefore it doesnt make sense to put them in translations

again think from the perspective of a translator that receives these files

they would just be like: "wtf theres an arrow thing here."

Reply:
correct but &#8592; and &#8594; are different and they are not being flipped.

Reply:
I'll try to fix the issue